bakery/README.md

```diff
@@ -5,9 +5,11 @@
 - Ensure `bakery/` is your working directory.
 - Run `yarn install` or `yarn`.
 
-### Generate a pipeline file
-- Run `yarn build [path/to/output]`
+### Generate a pipeline file for a particular environment
```

Comment:
Might want to add what environments are available. In a naive way i typed in `yarn build:production` and got an error.

```
error Command "build:production" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
You could also add a note to look in the env files or package.json to see what environments exist and the commands that are available.